EtharInc/EtharImageTracking
# Add basic remote image loading functionality

I've introduced basic remote image loading functionality into the project. The script titled "DynamicLibrary" is based off of an ARFoundation sample project called [basic image tracking](https://github.com/Unity-Technologies/arfoundation-samples#basicimagetracking), but I've modified it to include a URL field that we can pull the image texture from.

Currently, the only image being loaded remotely is the Radio. Once PR #4 is approved and merged, I'll migrate Tesla too. Also each url must manually be added in the Unity editor, but this can be improved in future iterations.

Steps to test:
1. Build the app to an android device and open it. Point the camera at the Radio card, and the radio model should appear in the environment. 
2. No visible changes should occur from previous versions, but the image is now being pulled from an [imgur url](https://i.imgur.com/b1z2nVR.png)

Workflow to add new models and image links:
1. Inside of Unity, open SampleScene. Select the AR Session Origin object.
3. Go to the "Dynamic Library" component on the gameobject, and click the + to add a new image to "Images".
4. Enter the name of your object into the "Name" field. Make sure that this matches the exact name of your prefab corresponding to the image. In the URL field, paste the address of your image.
5. Leave the "Texture" and "Width" fields blank, as these will be overridden when the app is run.
